WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -430,13 +430,25 @@ public boolean addBlog(Blog blog) {
     }
 
     public List<Map<String, Object>> getBlogsBy(String byString, String[] extraFields) {
-        return getBlogsBy(byString, extraFields, 0);
+        return getBlogsBy(byString, extraFields, 0, true);
```

Comment:
Are you sure it's safe to assume every place that calls this wants to hide Jetpack blogs?